arrus/core/api/devices/us4r/Us4R.h

```diff
@@ -179,6 +179,34 @@ class Us4R : public DeviceWithComponents {
     */
     virtual void setActiveTermination(std::optional<uint16> value) = 0;
 
+    /**
+    * Enables AFE auto offset removal.
+    *
+    * See docs of arrus::devices:: for more information.
+    */
+    virtual void enableAfeAutoOffsetRemoval(void) = 0;
+
+    /**
+    * Disables AFE auto offset removal.
+    *
+    * See docs of arrus::devices:: for more information.
+    */
+    virtual void disableAfeAutoOffsetRemoval(void) = 0;
+
+    /**
+    * Sets AFE auto offset removal accumulator cycles number.
+    *
+    * See docs of arrus::devices:: for more information.
+    */
+    virtual void setAfeAutoOffsetRemovalCycles(uint16_t cycles) = 0;
```

Comment:
More details, what is the "cycle" "removal delay". What are the units for these values ([s], number of cycles, etc.)?

Reply:
What is the default number of cycles (i.e. value that will be used, when this method is not called)?

Reply:
Updated comments in Us4R.h. Number of cycles per setting is specified by AFE datasheet and default is 0 = 2047 cycles:
![obraz](https://user-images.githubusercontent.com/55283527/186590608-22d85faf-1eae-4258-a814-a90b5f4d378c.png)